### PR TITLE
All networks support for rpc url (for WalletConnect only)

### PR DESCRIPTION
**Pull request by IlyaVi**

### CODE REVIEW

File: src/lib/utils/onboardHelper.ts [hunk 1]
```diff
@@ -63,19 +64,19 @@ const initOnboardOptions = {
...
       id: NETWORKS.GOERLI,
       label: 'Goerli testnet',
       token: TOKEN_NAMES[NETWORKS.GOERLI],
-      rpcUrl: 'https://late-thrilling-arm.ethereum-holesky.quiknode.pro/b64c32d5e1b1664b4ed2de4faef610d2cf08ed26',
+      rpcUrl: GOERLI_RPC_URL,
     },
     {
       id: NETWORKS.HOLESKY,
       label: 'Holesky',
       token: 'ETH',
```
Comment (alexstolr) [inline src/lib/utils/onboardHelper.ts:78]:
```suggestion
      token: TOKEN_NAMES[NETWORKS.HOLESKY],
```

File: src/services/contracts.service.ts [hunk 2]
```diff
@@ -16,8 +21,18 @@ const initGetterContract = ({ provider, network }: { provider: any; network: Net
       console.warn('Getter contract already exists');
     } else {
       console.warn('Creating new getter contract');
-      // const ethProvider = new ethers.providers.Web3Provider(provider, 'any');
-      const ethProvider = new ethers.providers.JsonRpcProvider(RPC_URL);
+      let ethProvider;
```
Comment (alexstolr) [inline src/services/contracts.service.ts:24]: Please add the type of new ethers.providers.JsonRpcProvider().

File: src/services/contracts.service.ts [hunk 3]
```diff
@@ -50,7 +65,18 @@ const initTokenContract = ({ provider, network }: { provider: any; network: Netw
     } else {
       console.warn('Creating new token contract');
       const ethProvider = new ethers.providers.Web3Provider(provider, 'any');
-      const rpcProvider = new ethers.providers.JsonRpcProvider(RPC_URL);
+      let rpcProvider;
+      if (shouldUseRpcUrlFlag) {
+        if (network.networkId === HOLESKY_NETWORK_ID) {
```
Comment (alexstolr) [inline src/services/contracts.service.ts:70]: ditto

**Review by alexstolr — CHANGES_REQUESTED** [review]
some small comments

**Review by axelrod-blox — APPROVED** [review]

**Pull request merged**